src/RobotMap.h

```diff
@@ -22,7 +22,8 @@ class RobotMap {
 	  static const int kIDLeft1Wheel = 3;
 	  static const int kIDLeft2Wheel = 4;
 	//..........Intake..........
-	  static const int kIDIntakeMotor = 5;
+	  static const int kIDIntakeMotorTop = 5;
+	  static const int kIDIntakeMotorBottom = 8;
```

Comment:
The reason you're having trouble with setting the direction and control of the bottom intake motor is that the climber subsystem is trying to use the same motor!